pkg/proto/nbcontract/v1/utils.go

```diff
@@ -0,0 +1,289 @@
+/*
+Portions Copyright (c) Microsoft Corporation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// All the helper functions should be hosted by another public repo later. (e.g. agentbaker)
+// Helper functions in this file will be called by bootstrappers to populate nb contract payload.
+package nbcontractv1
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/Azure/agentbaker/pkg/agent"
+	"github.com/Azure/agentbaker/pkg/agent/datamodel"
+	"github.com/blang/semver"
+)
+
+const numInPair = 2
+
+// GetLoadBalancerSKI returns the LoadBalancerSku enum based on the input string.
+func GetLoadBalancerSKU(sku string) LoadBalancerConfig_LoadBalancerSku {
+	if strings.EqualFold(sku, "Standard") {
+		return LoadBalancerConfig_STANDARD
+	} else if strings.EqualFold(sku, "Basic") {
+		return LoadBalancerConfig_BASIC
+	}
+
+	return LoadBalancerConfig_UNSPECIFIED
+}
+
+// GetNetworkPluginType returns the NetworkPluginType enum based on the input string.
+func GetNetworkPluginType(networkPlugin string) NetworkPlugin {
+	if strings.EqualFold(networkPlugin, "azure") {
+		return NetworkPlugin_NP_AZURE
+	} else if strings.EqualFold(networkPlugin, "kubenet") {
+		return NetworkPlugin_NP_KUBENET
+	}
+
+	return NetworkPlugin_NP_NONE
+}
+
+// GetNetworkPolicyType returns the NetworkPolicyType enum based on the input string.
+func GetNetworkPolicyType(networkPolicy string) NetworkPolicy {
+	if strings.EqualFold(networkPolicy, "azure") {
+		return NetworkPolicy_NPO_AZURE
+	} else if strings.EqualFold(networkPolicy, "calico") {
+		return NetworkPolicy_NPO_CALICO
+	}
+
+	return NetworkPolicy_NPO_NONE
+}
+
+// GetDefaultOutboundCommand returns a default outbound traffic command.
+func GetDefaultOutboundCommand() string {
+	return "curl -v --insecure --proxy-insecure https://mcr.microsoft.com/v2/"
+}
+
+func GetKubeletNodeLabels(agentPool *datamodel.AgentPoolProfile) map[string]string {
```

Comment:
Just to confirm, this function is not really used in the current PR, right? Is it supposed to be used by AKS-RP later?